components/feature/pwa/src/main/java/mozilla/components/feature/pwa/ManifestStorage.kt

```diff
@@ -16,7 +16,7 @@ import mozilla.components.feature.pwa.db.ManifestEntity
 /**
  * Disk storage for [WebAppManifest]. Other components use this class to reload a saved manifest.
  */
-class ManifestStorage(context: Context) {
+class ManifestStorage(context: Context, private val usedAtTimeout: Long = 2592000000) {
```

Comment:
Let's add some constant like `MILLISECONDS_IN_A_DAY` then set this to be `MILLISECONDS_IN_A_DAY * 30`, so its clear what this number represents.

It would also be good to have some doc comments that indicate this is in milliseconds and what its used for.
```
@param usedAtTimeout ...
```